java/ql/src/Security/CWE/CWE-789/UndoundedAllocationDeserialization.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC
```

Comment:
This `qhelp` file is missing from https://github.com/github/semmle-code/actions/runs/342049752 because of the typo in the file name. This seems like something that should result in a check failure.

It looks as if the file should be: `UnboundedAllocationDeserialization.qhelp